grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/db/BankDao.java

```diff
@@ -78,6 +71,14 @@ public void insertAll(Set<Bank> banks) {
     db.insertAll(queries);
   }
 
+  private Set<Bank> getQueryHandler(String query){
```

Comment:
Getting there, but here is the naming thing again. This isn't getting a QueryHandler, so it should be something like getBanks(String query).